src/vsphere_cpi/spec/integration/nsxt_spec.rb

```diff
@@ -306,6 +377,29 @@ def nsgroup_effective_logical_port_member_ids(nsgroup)
     results.map { |member| member.target_id }
   end
 
+  def create_server_pool(pool_name, nsgroup=nil)
+    server_pool = NSXT::LbPool.new(display_name: pool_name)
+    if nsgroup
+      resource = NSXT::ResourceReference.new(target_id: nsgroup.id, target_type: nsgroup.resource_type, target_display_name: nsgroup.display_name)
+      server_pool.member_group = NSXT::PoolMemberGroup.new(grouping_object: resource, max_ip_list_size: 2)
+    end
+    services_svc.create_load_balancer_pool(server_pool)
+  end
+
+  def verify_pool_member(server_pool, ip_address, port_no)
```

Comment:
Can we get rid of this function and just do something like:

```ruby
expect(...).to include(...)
```

Putting assertions in a function makes them a lot more opaque when reading the test.